drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -2191,6 +2191,11 @@ matterGeneric:
     deviceTypes:
       - id: 0x010B # Dimmable Plug-in Unit
     deviceProfileName: plug-level
+  - id: "matter/dimmer/switch"
+    deviceLabel: Matter Dimmer Switch
+    deviceTypes:
+      - id: 0x0104 # Dimmer Switch
```

Comment:
Dimmer Switch only has implements the client side for the On/Off and Level Control clusters so it won't be possible for us to control those clusters since the hub is the client and it needs server clusters on the device in order to communicate with it. Therefore I do not think it makes sense to add this fingerprint.